webpack.config.js

```diff
@@ -5,6 +5,11 @@ var definePlugin = new webpack.DefinePlugin({
     __DEBUG__: JSON.stringify(process.env.NODE_ENV === 'development'),
 });
 
+var providePlugin = new webpack.ProvidePlugin({
```

Comment:
Should we do the same for the promise and fetch polyfills? Or should this be changed to just a new entry in the `itowns` entry point?